include/SDL_mixer.h

```diff
@@ -457,6 +457,15 @@ extern DECLSPEC int SDLCALL Mix_OpenAudio(int frequency, Uint16 format, int chan
  */
 extern DECLSPEC int SDLCALL Mix_OpenAudioDevice(int frequency, Uint16 format, int channels, int chunksize, const char* device, int allowed_changes);
 
+/**
+ * Suspend or resume the whole audio output.
+ *
+ * \since This function is available since MixerX 2.7.0.
+ *
+ * @param pause_on Set audio output to pause
```

Comment:
This is unintelligible (at least to me.) You need something like `1 (for pause) or 0 (for resume)`, or something better.

Reply:
Done.